.github/workflows/release-stable.yml

```diff
@@ -91,6 +104,12 @@ jobs:
           TAG_NAME: ${{ steps.createTagName.outputs.TAG_NAME }}
         run: git push --atomic --set-upstream origin HEAD $TAG_NAME
 
+      - name: Create GitHub PR
+        env:
+          RELEASE_VERSION: ${{ steps.removeSnapshotSuffix.outputs.RELEASE_VERSION }}
+          CHECKOUT_BRANCH_NAME: ${{ steps.checkedOutBranchName.outputs.CHECKOUT_BRANCH_NAME }}
```

Comment:
do we really need this checkedOutBranchName step? We could just use ${{ github.ref_name }} here directly.

Reply:
I was thinking maybe it would change after switching the branch, but we can try and see